tests/testdata/msticpyconfig.yaml

```diff
@@ -1,7 +1,8 @@
 QueryDefinitions:
+  Default: 'testdata'
   Custom:
-    - "tests/testdata"
-    - "tests"
+    - 'C:\\queries'
```

Comment:
This doesn't look very portable. Did you mean to change this permanently?

Reply:
I thought that using absolute paths would be easier for configuration however having had a rethink I realized this was unworkable in a few situations so I have gone back to relative paths.